phase2/ignition/vanilla/kubelet.service

```diff
@@ -15,6 +16,7 @@ ExecStart=/usr/bin/docker run \
         -v /var/run:/var/run:rw \
         -v /var/lib/docker/:/var/lib/docker:rw \
         -v /var/lib/kubelet/:/var/lib/kubelet:shared \
+        -v /var/log:/var/log:shared \
```

Comment:
FYI referring to this: https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/fluentd-elasticsearch/fluentd-es-image/td-agent.conf#L113

Those are the position files, which keep track of how far into a particular log file fluentd has ingested. If the container was deleted (ie for an upgrade) these files would be lost if they're not being stored on the host via a volume mount, resulting in fluentd re-ingesting logs that have already been ingested previously.